cmd/ccboc/main.go

```diff
@@ -121,6 +121,58 @@ func getCalculationID(c *cli.Context) error {
 	return nil
 }
 
+func getCalculationResult(c *cli.Context) error {
+	teff := c.Float64("teff")
+	logG := c.Float64("logG")
+
+	fmt.Println(globalConfig.APIURL + "/calculations/results?teff=" + fmt.Sprintf("%0.1f", teff) + "&logg=" + fmt.Sprintf("%0.1f", logG))
+
+	body, responseError, err := request("GET", globalConfig.APIURL+"/calculations/results?teff="+fmt.Sprintf("%0.1f", teff)+"&logg="+fmt.Sprintf("%0.1f", logG), bytes.NewBuffer(nil))
+	if err != nil {
+		logrus.WithError(err).Fatal("error while perfoming the http request")
+	} else if responseError != nil {
+		logrus.WithFields(logrus.Fields{"message": responseError.Message, "status_code": responseError.StatusCode}).Fatal("errors occurred")
+	}
+
+	fileName := fmt.Sprintf("%0.1f___%0.1f", teff, logG)
```

Comment:
We should be able to get the filename from the headers. Also the name is incorrect since the file is a tar.gz

Reply:
See https://github.com/vega-project/ccb-operator/blob/master/cmd/apiserver/server.go#L209

Reply:
Getting error: x509: certificate signed by unknown authority
Do I need some special authentication or?

Reply:
To login to the API server, you need to authenticate, but still, this is not related to the changes you are making in this PR. You can try to test it locally.